h/accounts/test/views_test.py

```diff
@@ -79,9 +80,14 @@ def mock_flash_function():
                                 return_value=None)
 
 
-@pytest.mark.usefixtures('routes_mapper')
 class TestAuthController(object):
 
+    @pytest.fixture(autouse=True)
+    def routes(self, config):
+        config.add_route('forgot_password', '/forgot')
+        config.add_route('index', '/index')
+        config.add_route('stream', '/stream')
```

Comment:
The reason I ask about autouse fixtures is that you _can_ do this with class-level fixtures:

```python
@pytest.mark.usefixtures('my_fixture')
class TestExample:

    @pytest.fixture
    def my_fixture(self):
        ...

    ...
```

In my mind that has the advantage that `usefixtures` can also be used to activate a fixture for all methods of a class when that fixture is defined outside of that class. For example, any test module that has a fixture that more than one of its classes can share, can define that fixture at module level.

Always using `usefixtures` to activate a fixture for all methods of a class, wherever the fixture is defined, seems more consistent than sometimes using `autouse` and sometimes `usefixtures`.

(Also I've already done it in the above style in some existing test classes.)